crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -330,6 +379,7 @@ impl ViewEye {
                 // matters if the user starts interacting half-way through the lerp,
                 // and even then it's not a big deal.
                 eye_up: self.eye_up.lerp(other.eye_up, t).normalize_or_zero(),
+                speed: self.compute_speed_lerp(other, t, bounding_boxes),
```

Comment:
yeah this is a bit of a weird situation. It's just as broken as what it's beeing done with `mode`. Might as well do the same and just take over `other.speed` here
iirc practically this is used for transitioning on camera double clicks. So I think it's fine to just simplify this